hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/bucket/ListBucketHandler.java

```diff
@@ -51,7 +51,7 @@
   private int maxBuckets;
 
   @Option(names = {"--start", "-s"},
-      description = "The first bucket to start the listing")
+      description = "The listing will start from bucket after the startBucket.")
```

Comment:
Suggestion: The bucket to start the listing from. This will be excluded from the result.

Reply:
Thanks @nandakumar131 for the review.
Updated.